src/kernel/include/shared/cos_types.h

```diff
@@ -840,8 +848,8 @@ static inline void
 cos_mem_fence(void)
 { __asm__ __volatile__("mfence" ::: "memory"); }
 
-// ncpu * 16 (or max 256) entries. can be increased if necessary.
-#define COS_THD_INIT_REGION_SIZE (((NUM_CPU*16) > (1<<8)) ? (1<<8) : (NUM_CPU*16))
+/* 256 entries. can be increased if necessary */
+#define COS_THD_INIT_REGION_SIZE (1<<8)
```

Comment:
I guess I should put it back. I don't have a good/valid reason for removing it, except for changing the constant there from 16 to 32/64.